app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/api/ApiGeneratorHelper.java

```diff
@@ -145,6 +156,10 @@ public static Integration updateFlowsAndStartAndEndDataShapes(final Integration
             .resources(given.getResources()).build();
     }
 
+    private static Predicate<Flow> operationIdIs(String operationId) {
```

Comment:
how about renaming to `operationIdEquals` I find the current method name hard to read (because of `IdIs`) and initially thought that this is a typo 😄